string camera_id
panda_vision/BlockPose block_pose